addons/l10n_es/data/account_tax_data.xml

```diff
@@ -1609,6 +1609,44 @@
             }),
         ]"/>
     </record>
+    <record id="account_tax_template_p_iva0_bc" model="account.tax.template">
```

Comment:
This XML-ID is already used: https://github.com/odoo/odoo/blob/a597c54b5257e6388437e41527826ec8108f564e/addons/l10n_es/data/account_tax_data.xml#L2304